python/cugraph/cugraph/structure/graph_implementation/simpleDistributedGraph.py

```diff
@@ -323,10 +329,7 @@ def __from_edgelist(
             ddf, _client, return_type="dict"
         )
         del ddf
```

Comment:
We should not `persist` these dataframes anymore.  We used to `persist` to prevent `edata` computation.